node_cli/main.js

```diff
@@ -0,0 +1,85 @@
+const inquirer = require("inquirer");
+const ora = require("ora");
+
+const tasks = require("./tasks");
+const { logout } = require("./realm/realmAuth");
+const index = require("./index");
+const watch = require("./watch");
+
+async function mainMenu(authedUser) {
+  try {
+    const answers = await inquirer.prompt({
+      type: "rawlist",
+      name: "mainMenu",
+      message: "What would you like to do?",
+      choices: [
+        "Create a task",
+        "Show all of my tasks",
+        "Get a specific task",
+        "Change a task status",
+        "Edit a task",
+        "Delete a task",
+        "Watch for changes",
+        "Log out / Quit",
```

Comment:
It seems like this could lead to annoying bugs if the choice list and the switch statement below get out of sync. Why not put these choices in local constants above?

Reply:
Good idea. it was bothering me, too, that `inquirer` didn't offer an index for the selected item.